tests/test_asyncio/test_commands.py

```diff
@@ -114,7 +115,7 @@ async def test_acl_genpass(self, r: redis.Redis):
         password = await r.acl_genpass()
         assert isinstance(password, str)
 
-    @skip_if_server_version_lt(REDIS_6_VERSION)
+    @skip_if_server_version_lt(REDIS_6_2_VERSION)
```

Comment:
IMHO, maybe we should change REDIS_6_VERSION instead, to reflect a real release (6.0.0), and update tests associated here? I believe this was originally a placeholder we inherited when merging the projects.

It looks like [REDIS_6_VERSION](https://sourcegraph.com/github.com/redis/redis-py/-/blob/tests/test_asyncio/test_commands.py?L21:1&subtree=true#tab=references) is only used by these tests, meaning updating it, and the tests herein should go together given the helps docs (example: [ACL SETUSER](https://redis.io/commands/acl-setuser/)

WDYT?